.github/workflows/automation-scripts/update-changelogs.mjs

```diff
@@ -67,7 +67,12 @@ await $`mkdir ${incomingPath}`
 // find the latest release branch, and make that the target for the changelogs
 let targetBranch = await $`git branch -a --list "origin/release/[0-9]*.[0-9]*.x" | tail -n1 | sed 's/  remotes\\///'`;
 let currentBranch = await $`git branch --show-current`;
-let commitMessage = await $`git log --format=%B -n 1`;
+
+// the version in the commit message can be extracted from the latest tag
+const latestTag = await $`git describe --tags $(git rev-list --tags --max-count=1)`;
+const tag = latestTag.stdout.trim();
+const versionMatch = tag.match(/release\/(\d+\.\d+\.\d+)/);
+let commitMessage = versionMatch ? versionMatch[1] : null;
```

Comment:
So if this is the version of the branch based on the git tag, if we first run this workflow on release/4.7.x then it will get the tags of the current version and commit to master the new logs. However would this work if we were to do a back port on an older minor? So for example if this were on 3.x it would get the tag from there, then push changelogs to release/4.7.x, and then it would run again on release/4.7.x and I think get the tags for 4.x. So maybe using tags won't work here, or perhaps we can use a combination of tags and something else to get the message? I only thought of this based on our discussion on triggers yesterday